fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -3913,13 +3823,6 @@ public void testJsonLdProfileFlattened() throws IOException {
         assertEquals("Should be two values!", 2, titles.findValues("@value").size());
     }
 
-    @Test
-@Ignore
-    public void testPathWithEmptySegment() {
```

Comment:
Jersey seems to clean these up as the `@PathParam externalPath;` does not show the double slash.

Reply:
Actually, the HttpClient is where the clean-up happens. We should keep this test with the addition of some configuration like: https://github.com/fcrepo/fcrepo/pull/1798#discussion_r521773957